controller/rpc/pb/controller.proto

```diff
@@ -38,8 +40,9 @@ message Volume {
     string endpoint = 3;
     string frontend = 4;
     string frontendState = 5;
-    bool isRestoring = 6;
-    string lastRestored = 7;
+    bool isPurging = 6;
```

Comment:
Add the new variable to the end. Same for the other places.

Reply:
Didn't see the notification for the review until now. Will fix variable ordering.